### ISSUE TITLE
A11y_MySQLextensionforAzureDataStudio_New deployment Azure Database for MySQL_Name Role Value:Learn more link in manage dashboard page is not descriptive

### ISSUE THREAD

**Issue by VijayaBharathyA** [microsoft/azuredatastudio-mysql#188]
"[Check out Accessibility Insights! ](https://accessibilityinsights.io/)- Identify accessibility bugs before check-in and make bug fixing faster and easier.”
### GitHub Tags:
#A11yMAS; #A11yTCS; #A11ySev2; #DesktopApp; #GH-MySQLextensionforAzureDataStudio-Win32-Mar23; #MySQL extension for Azure Data Studio; #Win32; #WCAG4.1.2; #Name Role Value; #Screen Reader;  #Benchmark; #AILimited;  #NVDA; #JAWS;
#External; #External:AzureDataStudio; #Linked:BUG22656;
**"Please do not close this bug. This bug should only be closed by TCS, C+AI Accessibility or the Divisional Driver."**
### Environment and OS details:
Azure Data Studio Version: 1.42.0 (user setup)
Windows 11 Enterprise Version	22H2
Installed on	‎28-‎12-‎2022
OS build	22621.1413
Experience	Windows Feature Experience Pack 1000.22639.1000.0
### Pre-Requisite:

1. Install Azure data studio From here https://learn.microsoft.com/en-us/sql/azure-data-studio/download-azure-data-studio?view=sql-server-ver16&tabs=redhat-install%2Credhat-uninstall
2. Once installation done , Install MySQL Extension in Azure data studio
3. Tab Till  new connection and press enter 
4. Enter a valid server name username and password and press connect
### Repro Steps:

1. Launch Azure Data Studio.
2. Select the Connections icon (if not already selected) from left bar.
3. Click on the ellipses '...' and select "New Deployment"
4. In the Deployment options, select "Azure Database For MySQL" card.
5. Tab till Microsoft azure terms of use and Microsoft Privacy Statement
6. Verify that Learn more Microsoft azure terms of use and Microsoft Privacy Statement  defined as link or not
### Actual Result:
Learn more link in manage dashboard page is not descriptive.

### Expected Result:
Learn more link in manage dashboard page should be descriptive.
### Attachment:
![Learn more  link is not descriptive in Deployment Pre-Requisites page](https://user-images.githubusercontent.com/100189407/228879641-afd62439-ebe9-4638-a2de-d067dfb09921.png)
![Screenshot (51)](https://user-images.githubusercontent.com/100189407/228880823-221480d4-d830-4efd-b50d-a099951bae6c.png)

![Screenshot (88)](https://user-images.githubusercontent.com/100189407/229814695-3d9904c3-3bbb-422c-85aa-b9c4c3cb1bd3.png)

**Notes :**

![Screenshot (79)](https://user-images.githubusercontent.com/100189407/229701638-056b8d4a-b8ab-4852-9257-b5e9114a6542.png)
![Screenshot (87)](https://user-images.githubusercontent.com/100189407/229814569-5c214840-d103-4006-97c9-060ed74a2439.png)

**Comment by nitishsharma25** [comment]
This issue is external to MySQL extension as it is a common feature of Azure Data Studio.

**Comment by VijayaBharathyA** [comment]
This bug has two issue
### Title issue :
"Learn more" link is not descriptive in Deployment Pre-Requisites page. This by Design as per  this bughttps://github.com/microsoft/azuredatastudio/issues/15807
### Note issue : 
Learn more link in manage dashboard page is not descriptive. for note issue we have created New External bug 
So i have changed the title 
**Tracking Bug ID :https://github.com/microsoft/azuredatastudio/issues/22656
Could You please Reopen the bug since We have follow External process for this bug We will act on this bug once external bug is fixed and closed. Please don't close the bug**

**Comment by VijayaBharathyA** [comment]
#Regressed:05-15-23; #Closed
### Verified issue in below environment :
Issue is fixed
Version: 1.45.0-insider (user setup)
Commit: 4329d04da3b86689556d4dceab7d9b3d5cde3f58
Date: 2023-05-15T05:30:26.887Z
VS Code: 1.70.0
Electron: 19.1.8
Chromium: 102.0.5005.167
Node.js: 16.14.2
V8: 10.2.154.15-electron.0
OS: Windows_NT x64 10.0.22621
![Screenshot (153)](https://github.com/microsoft/azuredatastudio/assets/100189407/5302f4fb-d042-46fa-9beb-cf94d359569d)
![Screenshot (152)](https://github.com/microsoft/azuredatastudio/assets/100189407/c717d227-400a-41cc-8c64-52a049517474)
